addOns/pscanrules/CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ All notable changes to this add-on will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
+### Changed 
+- Update Vulnerabilities' references to use https links (Issue 8262).
```

Comment:
This should go in the existing Unreleased Changed section, and reference the specific rules being changed (the text you copied from the commonlib PR referred to a generic resource file in which all resource links were checked/updated).